foundation/donate.dd

```diff
@@ -47,6 +47,13 @@ the art in the D programming language.)
 
 $(H3 How can I donate?)
 
+$(DONATE_ITEM Donate through GitHub Sponsors, github,
+
+    The D Language Foundation is enrolled in the GitHub Sponors program. One-time donations here will go toward our general expense fund. We aim to use
```

Comment:
You too can be a Sponors!

Reply:
Anyone landing on the donate page is likely ready to hand over some money, probably because of a link that already sold them. We should simply provide the information they're looking for not repeat the sales pitch.

Reply:
Can I be a Sponor? What is a Sponor? :-)

Reply:
Ah. Got it.